Doc/howto/clinic.rst

```diff
@@ -193,6 +193,71 @@ The CLI supports the following options:
    The list of files to process.
 
 
+.. _clinic-classes:
+
+Classes for extending Argument Clinic
+-------------------------------------
+
+.. module:: clinic
+
+.. class:: CConverter
```

Comment:
Is this an actual class that I can import and use?  If not, should it get a `:noindex:`?

I was going through the howto and saw an example that points to [`Modules/zlibmodule.c`](https://github.com/python/cpython/tree/3.11/Modules/zlibmodule.c), but in the source there's no mention of `CConverter`.

Reply:
You must subclass the CConverter class when you write custom converters. Perhaps we can point to `Modules/posixmodule.c` instead; there's a lot of custom converters there.

Reply:
Answering your first question; you cannot import it, but you can use it. It is "magically" there when you write a `[python input]` block.

Reply:
I'm not sure about the `:noindex:`, though.